ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -44,4 +44,5 @@ bash -c ' \
     yum install -y python-pip ; \
     pip install -U remoto ; \
     yum remove -y python-pip ; \
+    yum install -y openssh-clients ; \
```

Comment:
I think we can combined the openssh-clients package installation with python-pip instead of using yum install twice

Reply:
fixed